Makefile

```diff
@@ -28,7 +28,7 @@ ifeq ($(BUILD_IMAGE_NAME),)
 BUILD_IMAGE_NAME=otelopscol-build
 endif
 
-OTELCOL_BINARY=google-cloudops-opentelemetry-collector
+OTELCOL_BINARY=google-cloudops-opentelemetry-collector_$(GOOS)_$(GOARCH)$(EXTENSION)
```

Comment:
Same question as here: https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/6#discussion_r463359067

Reply:
I just tried it. First of all, it runs the build command with `windows`, and the final output in the dist folder is `google-cloudops-opentelemetry-collector.x86_64.0.0.0.goo`

Reply:
Ok cool all good then